### PR TITLE
Fix bucket

### PR DESCRIPTION
**Pull request by connoravo**

### CODE REVIEW

**Review by Mark0930 — DISMISSED** [review]

**Review by dan-hill2802 — COMMENTED** [review]
this is the wrong bucket, both the before and after values are wrong

File: batch_object_tagger.tf [hunk 1]
```diff
@@ -39,6 +39,7 @@ data "aws_iam_policy_document" "pdm_object_tagger" {
 
     resources = [
       "${data.terraform_remote_state.common.outputs.config_bucket.arn}/*",
+      "${data.terraform_remote_state.common.outputs.published_bucket.arn}/*"
```
Comment (dan-hill2802) [inline batch_object_tagger.tf]: you need to split this statement, one for the csv access and one for the published data access. they require different permissions. You also need a 3rd statement for the configs KMS key (output from common)

**Review by Mark0930 — DISMISSED** [review]

**Review by dan-hill2802 — CHANGES_REQUESTED** [review]
need to fix IAM

File: batch_object_tagger.tf [hunk 2]
```diff
@@ -39,9 +39,53 @@ data "aws_iam_policy_document" "pdm_object_tagger" {
...
+
+    actions = [
+      "kms:Decrypt",
+      "kms:DescribeKey",
+    ]
+
+    resources = [
+      "${data.terraform_remote_state.common.outputs.published_bucket.arn}/*}",
+      ]
+  }
```
Comment (dan-hill2802) [inline batch_object_tagger.tf]: it has no need to access the data here, just tag it, so it shouldn't need decrypt or getobject

Comment (connoravo) [reply batch_object_tagger.tf]: GetObject required in order to retrieve metadata: https://docs.aws.amazon.com/AmazonS3/latest/API/API_HeadObject.html

**Pull request closed without merging**